src/amo/components/RatingManager/index.js

```diff
@@ -18,34 +21,54 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import DefaultRating from 'ui/components/Rating';
+/* eslint-disable no-duplicate-imports */
```

Comment:
Whoa, hah. We were already using `import/no-duplicates` so we didn't even need the core `no-duplicate-imports`. That was confusing. Fixed, thanks!